### PR TITLE
Improve the use of documents

### PR DESCRIPTION
**Pull request by zhiqiang-bianjie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,104 +1,6 @@
...
-    - Undelegate transaction construction and signature
-    - Redelegate transaction construction and signature
-    - SetWithdrawAddress transaction construction and signature
-    - WithdrawDelegatorReward transaction construction and signature
-    - WithdrawValidatorCommission transaction construction and signature
-    
-In addition, IRISnet also supports hot and cold wallets, offline signatures of transactions by scanning code.
-## Installation
-
+It mainly provides account generation, transaction construction and offline signature functions of [irisnet](./docs/IRIS.md) and [cosmos](./docs/ATOM.md) You can install this library in the following ways
```
Comment (zhangyelong) [inline README.md]: offline signature -> offline signing

File: docs/ATOM.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
+# ATOM
+
+Irisnet-crypto has completed the docking cosmos main network(chain-id:cosmoshub-2),supports signatures for the following transactions
```
Comment (zhangyelong) [inline docs/ATOM.md]: supports signatures -> supports signing

File: docs/ATOM.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  "gas": 10000,
+  "memo": "",
+  "type": "",
+  "msg": msg
+}
+```
+The parameters are as follows:
+- chain_id : blockchain's chain_id
+- from : Signer's address of the transaction
+- account_number : you can get it from lcd's 'auth/accounts/ api'
```
Comment (zhangyelong) [inline docs/ATOM.md]: you can get it from lcd api "/auth/accounts/"

File: docs/ATOM.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+- sequence : you can get it from lcd's 'auth/accounts/ api'
```
Comment (zhangyelong) [inline docs/ATOM.md]: you can get it from lcd api "/auth/accounts/"

File: docs/ATOM.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+- memo : transaction description(Up to 100 characters)
+- type : transaction type,enumeration values are as follows
+    - transfer
+    - delegate
+    - undelegate
+    - begin_redelegate
+    - set_withdraw_address
+    - withdraw_delegation_reward
+    - withdraw_validator_commission
+- msg : the specific transaction content, different transactions, and data structure are also different. For details, please refer to the following example.
```
Comment (zhangyelong) [inline docs/ATOM.md]: different transactions, and data structure are also different. 
->
different type of transaction has different structure

File: docs/ATOM.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+    - withdraw_delegation_reward
+        ```json
+        {
+            "validator_addr": "cosmosvaloper1r33tlrh7cawtzxmct7zatau6vdydp0rg3ywegm"
+        }
+        ```
+    - withdraw_validator_commission(don't need)
+    
+    
+When you have constructed the request parameters, you can call crypto's signature method.
```
Comment (zhangyelong) [inline docs/ATOM.md]: signature method -> signing method

File: docs/ATOM.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+    ```
+    *getBuilder* has one parameter,used to specify which blockchain transaction to construct,we choose 'cosmos'
+
+3. Construct and sign a transaction
+    ```js
+    let stdTx = builder.buildAndSignTx(request,privateKeyey);
+    ```
+    *buildAndSignTx* has two parameters:
+      - request : first mentioned above
+      - privateKeyey : your private key
```
Comment (zhangyelong) [inline docs/ATOM.md]: privateKeyey -> privateKey

File: docs/IRIS.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
+# IRIS
+
+Irisnet-crypto has completed the docking irisnet main network(chain-id:irishub)， supports signatures for the following transactions
```
Comment (zhangyelong) [inline docs/IRIS.md]: supports signatures -> supports signing

File: docs/IRIS.md [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+  "gas": 10000,
+  "memo": "",
+  "type": "",
+  "msg": msg
+}
+```
+The parameters are as follows:
+- chain_id : blockchain's chain_id
+- from : Signer's address of the transaction
+- account_number : you can get it from lcd's 'auth/accounts/ api'
```
Comment (zhangyelong) [inline docs/IRIS.md]: same as docs/ATOM.md

File: docs/IRIS.md [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+- sequence : you can get it from lcd's 'auth/accounts/ api'
```
Comment (zhangyelong) [inline docs/IRIS.md]: same as docs/ATOM.md

File: docs/IRIS.md [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+- gas : gas limit(20000)
+- memo : transaction description(up to 100 characters)
+- type : transaction type,enumeration values are as follows
+    - transfer
+    - delegate
+    - begin_unbonding
+    - redelegate
+    - withdraw_delegation_rewards_all
+    - withdraw_delegation_reward
+- msg : the specific transaction content, different transactions, and data structure are also different. For details, please refer to the following example.
```
Comment (zhangyelong) [inline docs/IRIS.md]: same as docs/ATOM.md

File: docs/IRIS.md [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+        ```
+    
+    - withdraw_delegation_rewards_all(don't need)
+    - withdraw_delegation_reward
+        ```json
+        {
+            "validator_addr": "fva1kca5vw7r2k72d5zy0demszmrhdz4dp8t4uat0c"
+        }
+        ```
+When you have constructed the request parameters, you can call crypto's signature method.
```
Comment (zhangyelong) [inline docs/IRIS.md]: signature -> signing

File: docs/ATOM.md [hunk 13]
```diff
@@ -0,0 +1,139 @@
...
+        {
+          "validator_addr": "cosmosvaloper1r33tlrh7cawtzxmct7zatau6vdydp0rg3ywegm",
+           "amount":{
+               "denom": "stake",
+               "amount": "10"
+           }
+        }
+        ```
+    
+    - undelegate
```
Comment (shineabel) [inline docs/ATOM.md]: undelegate和unbond统一起来，统一用undelegate

**Pull request merged**